src/clients/api/queries/useGetTreasuryTotals.ts

```diff
@@ -0,0 +1,113 @@
+import { useMemo } from 'react';
+import BigNumber from 'bignumber.js';
+import { useGetMarkets, useGetVTokenBalancesAll, IGetVTokenBalancesAllOutput } from 'clients/api';
+import { TREASURY_ADDRESS } from 'config';
+import { VBEP_TOKENS } from 'constants/tokens';
+import { Market } from 'types';
+import { indexBy } from 'utilities/common';
+
+export interface IData {
+  treasuryTotalSupplyUsdBalanceCents: BigNumber;
+  treasuryTotalBorrowUsdBalanceCents: BigNumber;
+  treasuryTotalUsdBalanceCents: BigNumber;
+  treasuryTotalAvailableLiquidityUsdBalanceCents: BigNumber;
```

Comment:
Same as last comment:
```suggestion
  treasuryTotalSupplyBalanceCents: BigNumber;
  treasuryTotalBorrowBalanceCents: BigNumber;
  treasuryTotalBalanceCents: BigNumber;
  treasuryTotalAvailableLiquidityBalanceCents: BigNumber;
```